test/merge_test.js

```diff
@@ -0,0 +1,270 @@
+goog.require('helper');
+
+/**
+ * @fileoverview Unit tests for merge().
```

Comment:
Nit: File overview should go above require statements.

https://google.github.io/styleguide/jsguide.html#source-file-structure

Reply:
Moved